tutorials/plot_02-FOOOF.py

```diff
@@ -43,7 +43,7 @@
 # the FOOOF module.
 #
 # The model is broadly agnostic to exactly how power spectra are computed. Common
-# methods, such as using Welch's method to compute the spectrogram
+# methods, such as Welch's method, can be used to compute the spectrogram.
```

Comment:
So, actually I think in this case, `spectrogram` should really say `spectrum` (my error). Can you edit this sentence to say spectrum instead of spectrogram!